src/ResourceManager/Sql/Commands.Sql/Database/Model/AzureSqlDatabaseModel.cs

```diff
@@ -138,6 +129,17 @@ public class AzureSqlDatabaseModel
         /// </summary>
         public bool? ZoneRedundant { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Sku of the database
+        /// </summary>
+        public Sku Sku { get; set; }
```

Comment:
I think model properties are not supposed to be complex types like this. Cormac?

Reply:
Sounds good as long as it looks good in the console